lms/templates/design-templates/header/_trinity-header.html

```diff
@@ -0,0 +1,159 @@
+ <%namespace name='static' file='/static_content.html'/>
+
+ <%
+   import third_party_auth
+   from third_party_auth import pipeline
+   try:
+     tpa_idp = third_party_auth.provider.Registry.accepting_logins()[0]
+     custom_login_url = pipeline.get_login_url(tpa_idp.provider_id, pipeline.AUTH_ENTRY_REGISTER, redirect_url=request.path)
+   except:
+     custom_login_url = '/login'
+
+   custom_register_url = 'https://pass-staging.texasgateway.org/register/'
+   custom_profile_url = 'https://pass-staging.texasgateway.org/profile_edit'
+   custom_logout_url = 'https://pass-staging.texasgateway.org/oauth/v2/auth/logout'
+ %>
+
+ ## mako
+
+ <!-- top navigation -->
+ <section class="a--header-design-01 a--site-nav">
+   <div class="bs-container a--container a--site-nav__container">
+     <div class="a--site-nav__group-left">
+       <a href="${logo_url}" class="a--site-nav__logo__container a--site-nav__logo__positive">
+         <img class="a--site-nav__logo a--logo-full" src="${logo_positive}">
+       </a>
+       <a href="${logo_url}" class="a--site-nav__logo__container a--site-nav__logo__negative">
+         <img class="a--site-nav__logo a--logo-full" src="${logo_negative}">
+       </a>
+       % if display_course_name:
+       <div class="a--site-nav__course-name">
+         <span>${course_number}</span>${course_name}
+       </div>
+       % endif
+     </div>
+     <nav class="a--site-nav__group-right" id="site-menu">
+       <ul class="a--site-menu__container">
+         % if user_authenticated:
+           <li class="a--site-menu__group">
+             <ul>
+               % for menu_item in get_header_menu_logged_in_extra_items():
+                 <li class="a--${menu_item['special_class']}">
+                   <a href="${menu_item['link_URL']}">${_(menu_item['link_title'])}</a>
+                 </li>
+               % endfor
+             </ul>
+           </li>
+           <li class="a--site-menu__group">
+             <ul>
+               <li>
+                 <a href="/courses">${_("Find Courses")}</a>
+               </li>
+             </ul>
+           </li>
+         % else:
+           <li class="a--site-menu__group">
+             <ul>
+               % for menu_item in get_header_menu_logged_out_extra_items():
+                 <li class="a--${menu_item['special_class']}">
+                   <a href="${menu_item['link_URL']}">${_(menu_item['link_title'])}</a>
+                 </li>
+               % endfor
+             </ul>
+           </li>
+           <li class="a--site-menu__group">
+             <ul>
+               <li>
+                 <a href="/courses">${_("Find Courses")}</a>
+               </li>
+             </ul>
+           </li>
+           <li class="a--site-menu__group">
+             <ul>
+               <li class="a--nav-cta">
+                 <a href="${custom_login_url}">Trinity Account</a>
```

Comment:
I think for now the button text should just login, they haven't requested any special text, and they don't use trinity as name I think. But is a good question to ask them.